docs/reference/frozen-indices.asciidoc

```diff
@@ -0,0 +1,79 @@
+[role="xpack"]
```

Comment:
@lcawl any pointers how and where we wire this in are appreciated!

Reply:
In my opinion, this content can all stay in the Elasticsearch Reference.  To that end, I submitted a couple of commits to make it work.  You can preview it here: https://doc-review-63bf6.firebaseapp.com/es2/frozen-indices.html